src/wazuh.py

```diff
@@ -111,6 +111,8 @@ def _update_wazuh_configuration(  # pylint: disable=too-many-locals
         _generate_cluster_snippet(node_name, node_type, master_address, cluster_key)
     )
     elements[0].append(new_cluster)
+    syslog = etree.fromstring(_generate_syslog_snippet())  # nosec
```

Comment:
May I ask what the # nosec might be for?
Perhaps `# nosec: <CVE-number>` might also be better with a comment :D

Reply:
It looks like that's not considered unsafe anymore with the new lxml version so I'm removing it. Thanks!